### ISSUE TITLE
Typescript: possible to get url for particular method from schema

### ISSUE THREAD

**Issue by MirKml** [RicoSuter/NSwag#2373]
tl;dr; There is no possibility to get particular url from generated Typescript client. Only methods are available.

Use case:
We have the OpenAPI 3 schema as main source of information and Typescript client is generated from these one. In frontend app, we use this generated client. In Api, there is a GET method for getting image

```
/showImage/{id}:
    get:
      summary: Returns image with particular id.
      parameters:
        - in: path
          name: id
          description: image id
          required: true
          schema:
            type: integer
            format: int32
      responses:
        '200':
          content:
            image/*:
              schema:
                type: string
                format: binary
        '404':
          description: Image with specified id was not found
```

This 'method' will be used in frontend as url for src attribute of `<img src='' />`.  There is no possibility to get `/showImage/{id}` url from generated Typescript code. Only `client.showImage(id: string)` method exists, but this isn't what we want.

Do you know about any solution except hard code url directly in src attribute? E.g. generate methods e.g. `client.getUrlForShowImage(id)` into the Typescript. These can be later used in `client.showImage(id: string)` method. It means changes somewhere in liquid templates I mean.

**Comment by devbrains-com** [comment]
Same issue here. It would be nice, to have a possibility to get the Endpoint URL instead of the response of the request for FileContents.

Is there any possibility?
